tests/integration/conftest.py

```diff
@@ -40,7 +40,7 @@ async def deploy(ops_test: OpsTest):
             apps=[NGINX_NAME, POSTGRES_NAME, REDIS_NAME],
             status="active",
             raise_on_blocked=False,
-            timeout=600,
+            timeout=2000,
```

Comment:
I am assuming this is intentional, last time it was not.

General question, is a timeout best we can do for integration tests? One would think there would be an event monitoring mechanism.

Reply:
Yes this was intentional, last time it was a sleep, this is a timeout on a wait_for_idle which is different. Unfortunately this is due to the redis charm requiring some time to settle, the data platform team have suggested this is a more reasonable timeout than what I had originally. The event monitoring is the status reaching `active` this is what is deciding when to progress to the next step. The timeout is just the allocated time allowance for this to happen. The need for this to be longer is largely due to the low resources of the github runners.